src/helix-ui/styles/core/hx-popover.less

```diff
@@ -0,0 +1,46 @@
+hx-popover {
+  background-color: @gray-0;
+  border-color: @gray-300;
+  border-style: solid;
+  border-width: 1px;
```

Comment:
can we combine border-color, border-style, and border-width into 1 line?

Reply:
good catch